README.md

```diff
@@ -1,80 +1,133 @@
-# Bulldozer
+# bulldozer
 
-The `Bulldozer` is a bot that auto-merges PRs when all status checks are green and the PR is reviewed.
-It uses [GitHub status checks](https://developer.github.com/v3/repos/statuses/) and [GitHub reviews](https://developer.github.com/v3/pulls/reviews/) to do this.
+`bulldozer` is a [GitHub App](https://developer.github.com/apps/) that auto-merges
+PRs when all status checks are green and the PR is reviewed
 
+## Configuration
 
-## Running the server
+By default, the behavior of the bot is configured by a `.bulldozer.yml` file at
+the root of the repository. The file name and location are configurable when
+running your own instance of the server.
 
-* The server expects a config file name to be passed-in with the `-c` parameter (see `./bulldozer --help`).  This config file should look like this.
+- If the file does not exist, `bulldozer` will attempt to read from the configuration_v0_path, which is
```

Comment:
If we want to call out the `configuration_v0_path`, I'd put that in a section on migrating from 0.4.x to 1.x. I think it is confusing for it to appear this early.